Optimize (CQA) - Mng content - Managing CVs (#4897)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-composite-content-view-by-using-cli.adoc

= Creating a composite content view by using Hammer CLI

[role="_abstract"]

> Create a composite content view to merge repositories from multiple content views into a single host-facing repository.

Reviewer: same as above; Maybe we can review/finish the other PR first and then just apply the same suggestion here too.

Author: ```suggestion
Create a composite content view to combine repositories from multiple content views for your hosts to consume.
```
